src/main/java/com/atherys/quests/managers/LocationManager.java

```diff
@@ -14,12 +16,45 @@
 
 public final class LocationManager extends AbstractMongoDatabaseManager<LocationManager.QuestLocation> {
 
-    protected LocationManager() {
+    private static LocationManager instance = new LocationManager();
+    private static Gson gson = GsonUtils.getGson();
```

Comment:
Ideally, getGson would be called during initialization of the LocationManager. Having it as a static field introduces possible ndefined behaviors.